Cython/Compiler/Main.py

```diff
@@ -129,6 +129,14 @@ def process_pxd(self, source_desc, scope, module_name):
             result_sink = create_default_resultobj(source, self.options)
             pipeline = Pipeline.create_pyx_as_pxd_pipeline(self, result_sink)
             result = Pipeline.run_pipeline(pipeline, source)
+        elif Options.use_shared_utility:
```

Comment:
It seems a hack to put this here, especially based on the global option. Doesn't this even block any access to regularly cimported pxd files?

Reply:
I think all this does is enable our internal utility-code only `@cname` directive on all pxd files. So it doesn't block anything, but isn't ideal.